curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparisons-with-the-logical-or-operator.md

```diff
@@ -36,7 +36,7 @@ return "Yes";
 
 # --instructions--
 
-Combine the two `if` statements into one statement which returns `"Outside"` if `val` is not between `10` and `20`, inclusive. Otherwise, return `"Inside"`.
+Combine the two `if` statements into one statement which returns `Outside` if `val` is not between `10` and `20`, inclusive. Otherwise, return `Inside`.
```

Comment:
```suggestion
Combine the two `if` statements into one statement which returns the string `Outside` if `val` is not between `10` and `20`, inclusive. Otherwise, return the string `Inside`.
```